### DISCUSSION TITLE
How do I add a label to an issue?

### DISCUSSION THREAD

**Discussion by rajeevkumar504** [community/community#53473] [New to GitHub] [answered]
Hello,

I'm relatively new to GitHub and having trouble figuring out how to add a label to an issue in a repository. I've searched the documentation and looked through the UI, but I can't seem to find where to do this.

Can someone please explain how to add a label to an issue in a GitHub repository? I'd greatly appreciate any guidance or tips on this.

Thank you in advance for your help.

**Comment by NJacksonDev** [comment, marked as answer]
Below are some step by step instructions which should solve your issue.

1. Open the repository on GitHub and navigate to the Issues tab.
2. Click on the issue you want to add a label to.
3. On the right-hand side of the issue, you should see a section called "Labels". If there are already labels assigned to the issue, they will be displayed here.
4. To add a new label, click on the "Labels" dropdown and start typing the name of the label you want to add.
5. If the label already exists, it will appear in the dropdown list and you can select it. If it doesn't exist, you can create a new label by typing its name and pressing Enter.
6. Once you've selected the label(s) you want to add, click on the "Submit new label" button.

**Reply by rajeevkumar504** [reply]
I did the same but my right-hand side corner is unhighlighted like in this screenshort
<img width="410" alt="Screenshot 2023-04-23 at 11 05 36 AM" src="https://user-images.githubusercontent.com/97542658/233821684-3976095e-81ff-4e3d-95c2-bc4699767008.png">

**Reply by Limeload** [reply]
I have a similar issue, how to fix this when label and assign is disabled?

**Reply by ink-boy** [reply]
I too am seeing that.

**Reply by neverson42** [reply]
From the [github docs](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels):
![image](https://github.com/community/community/assets/35622011/00df8002-0825-4e6f-a27b-6552bb75a09b)

Only members with "triage access" can add tags.  This is essentially a forum moderator of the repository. Read more about roles [here](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).

@beRajeevKumar  @Limeload @Moazzem-Hossain-pixel @ink-boy

**Reply by Tekaiguy** [reply]
> Only members with "triage access" can add tags.

Keeping the gear icon as a greyed-out option would:

1. Let users know they're in the right place
2. Let users know they can't interact with this feature
3. Not add complexity over the current function (it's just formatting)

**Reply by rzy114514** [reply]
so I without "triage accesss" I cannot set label for my issue?

**Comment by NJacksonDev** [comment]
The label screen should look like the below if you click the "Labels" button on the immediately prior page.

On the label management page, you can create new labels, edit existing labels, and assign colors to each label. You can also delete labels that you no longer need.

You can also create new labels by clicking on the "Edit labels" button and typing in the name of the new label. You can assign a color to the label by selecting a color from the color palette.

![Screenshot 2023-04-23 at 11 18 00 AM](https://user-images.githubusercontent.com/12886956/233848281-0137b9d2-da67-40ef-a521-69c70a65a585.png)

**Comment by MaxFun** [comment]
Received email indicating I need to add a label but none of the above works or looks like my screen, see attached. 
![Label Problem](https://github.com/community/community/assets/3932952/f5fcdacb-f052-41da-b6a6-d63646f9a294)
. 
Any suggestions?

**Comment by daveycodez** [comment]
I don't see any way to edit or add labels to existing issues.

**Comment by hermannx5** [comment]
Also new to github...I also do not see any possibility to change/add labels ....

**Comment by miljacz** [comment]
Does anyone have an answer for this? It looks like this for me too. 
<img width="481" height="539" alt="image" src="https://github.com/user-attachments/assets/80dce14f-5a31-447a-9b58-1c90af8e2666" />

**Reply by CheesyMaxwell** [reply]
welp, idonno

**Reply by CheesyMaxwell** [reply]
@miljacz

**Comment by TariqKingCoder** [comment]
To add a label to an issue on GitHub, open the issue, look at the right‑hand sidebar, and find the Labels section; click the gear icon or “Labels” button, then select one or more labels from the list (or create a new one if you have permission), and they’ll be attached to the issue immediately.

**Comment by mubtashimrafid** [comment]
You can add a label to an issue from the issue page itself, but you need the proper permissions in the repository.

Here’s how to do it:

1. Open the issue you want to label.
2. On the right-hand sidebar, look for the **Labels** section.
3. Click the gear icon or **Labels** field.
4. Select one or more labels from the list.
5. The label will be added immediately to the issue.

If you do not see the Labels option, it may mean:

* You only have read access to the repository
* The repository owner has restricted issue management permissions
* The repository has no labels created yet

Repository maintainers can also create new labels under **Issues → Labels**.
